Pkgfile

```diff
@@ -178,6 +178,11 @@ vars:
   libnl_sha256: 9fe43ccbeeea72c653bdcf8c93332583135cda46a79507bfd0a483bb57f65939
   libnl_sha512: 80fbbc079299c90afd2a5eda62e4d4f98bf4ef23958c3ce5101f4ed4d81d783af733213bb3bab15f218555d8460bc2394898f909f4ac024fc27281faec86a041
 
+  # renovate: datasource=github-tags depName=SELinuxProject/selinux
```

Comment:
```suggestion
  # renovate: datasource=git-tags extractVersion=^libsepol-(?<version>.*)$  depName=https://github.com/SELinuxProject/selinux.git
```

Reply:
Tags like 3.7 plain should contain everything: https://github.com/SELinuxProject/selinux/releases/tag/3.7

Reply:
ahh then it's fine, let's keep renovate config json as is then https://github.com/siderolabs/tools/pull/379/files#diff-f82f7d36a61d22f640c25bdb8b0435bf9cb38679d99d5f460753fa668ee7cdb3R40 here